entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/extension/PropertyDescriptorExt.kt

```diff
@@ -33,40 +33,40 @@ val PropertyDescriptor.assignmentPsi: KtExpression
 
 private val propertyHintAnnotations: List<String> =
     listOf(
-        "godot.annotation.IntRange",
```

Comment:
alpha-sorted

Reply:
I'm not sure wether this is a good idea. It was sorted like in https://github.com/godotengine/godot/blob/master/core/object.h before

Reply:
I personnally prefer to folow object header order, it would be easier to maintain.